dnf5/include/dnf5/context.hpp

```diff
@@ -161,6 +161,9 @@ class DNF_API Context : public libdnf5::cli::session::Session {
     void set_json_output_requested(bool json_output);
     bool get_json_output_requested() const;
 
+    void set_create_repos(bool create_repos);
```

Comment:
This is an API. Please add a documentation string.

Reply:
Thanks, added.